internal/gcsx/append_object_creator_test.go

```diff
@@ -89,8 +89,9 @@ func (t *AppendObjectCreatorTest) SetUp(ti *TestInfo) {
 func (t *AppendObjectCreatorTest) call() (o *gcs.Object, err error) {
 	o, err = t.creator.Create(
 		t.ctx,
+		t.srcObject.Name,
 		&t.srcObject,
-		t.mtime,
+		&t.mtime,
```

Comment:
Shall we modify CallsComposeObjects() and CallsComposeObjectsWithObjectProperties() test in this file to use local mtime instead and assert that the received metadata gcsfuse_mtime is in UTC format?

Reply:
I didnt follow.  CallsComposeObjects() and CallsComposeObjectsWithObjectProperties() are already validating that mtime is being set correctly.

Reply:
We modified the Create method to use UTC time. However in the test we are still using this:
```
t.mtime = time.Now().Add(123 * time.Second).UTC()
// other code
ExpectEq(t.mtime.Format(time.RFC3339Nano), req.Metadata["gcsfuse_mtime"])
```

Shall we change this to:
```
t.mtime = time.Now().Add(123 * time.Second)
// other code
ExpectEq(t.mtime.UTC().Format(time.RFC3339Nano), req.Metadata["gcsfuse_mtime"])
```